smart-contracts/contracts/lst-dex-adapter-osmosis/.cargo/config.toml

```diff
@@ -3,6 +3,6 @@ wasm = "build --release --target wasm32-unknown-unknown"
 schema = "run --bin schema --features schema"
 lint = "clippy -- -D warnings"
 lint-fix = "clippy --fix -- -D warnings"
-unit-test = "test --lib"
+unit-test = "test"
 test-vault = "test --lib -p basic-vault"
```

Comment:
maybe this is wrong "basic-vault"

Reply:
yes, that was copy-pasta that I didn't clean up. Removed both test-vault and clip.